docs/accelerometer.rst

```diff
@@ -59,6 +59,22 @@ Functions
     Return a tuple of the gesture history. The most recent is listed last.
     Also clears the gesture history before returning.
 
+.. py:function:: get_rate()
+
+    Get the accelerometer samping rate, in Hz.
+
+.. py:function:: set_rate()
```

Comment:
I think we should remove the get/set rate methods, there is no such functionality in MakeCode